### PR TITLE
add unminified builds

### PR DESCRIPTION
**Pull request by DanielRuf**

### CODE REVIEW

File: src/stickybits.js [hunk 1]
```diff
@@ -186,7 +186,7 @@ Stickybits.prototype.computeScrollOffsets = function computeScrollOffsets (item)
   const parent = it.parent
   const isCustom = !this.isWin && p.positionVal === 'fixed'
   const isBottom = p.verticalPosition !== 'bottom'
-  const scrollElOffset = isCustom ? p.scrollEl.getBoundingClientRect().top : 0
+  const scrollElOffset = isCustom ? document.querySelector(p.scrollEl).getBoundingClientRect().top : 0
```
Comment (yowainwright) [inline src/stickybits.js]: ^ Nice catch! 😍 

Could this be in another Pull Request? 

Also, could this be on [line line 61](https://github.com/DanielRuf/stickybits/blob/08f63e94ac9e63fe17b58373f25c0f03c31dc57b/src/stickybits.js#L61)?

🙏🙏🙏

**Comment by yowainwright** [comment]
@DanielRuf THANK YOU SO MUCH FOR SNAGGING THESE ISSUES!

I should be able to help out over the weekend. Let me know if you would like me to take on your suggested change for the 'custom scroll element'.

File: src/stickybits.js (hunk 1 above)
Comment (DanielRuf) [reply src/stickybits.js]: Sure, will do it later today.

Comment (yowainwright) [reply src/stickybits.js]: @DanielRuf 🙏

**Comment by DanielRuf** [comment]
Or should I change the old scripts?

> NOTE: As of StickyBits 3.0.0, unminified versions are not built

**Comment by DanielRuf** [comment]
Unminified builds make sense, eg when we need the files in custom build environments where we use webpack and other solutions and only use parts of stickybits for example so treeshaking processes can work with it better.

**Comment by DanielRuf** [comment]
It is also easier for testing / debugging. The unminified builds always make sense imo.

**Comment by DanielRuf** [comment]
I've reverted the line change and created a new PR with the change #231

**Comment by yowainwright** [comment]
@DanielRuf your opinion is very valid and appreciated. 😁 

~Thanks for the PR!
